test/mpi/maint/coll_cvars.txt

```diff
@@ -331,6 +332,9 @@ algorithms:
                 .MPIR_CVAR_IBCAST_ALLGATHER_RECEXCH=3
             tsp_ring
                 .MPIR_CVAR_IBCAST_RING_CHUNK_SIZE=4096
+            posix:release_gather
+                .MPIR_CVAR_COLLECTIVE_FALLBACK=silent
```

Comment:
Can we be sure the algo is being tested if silent fallback is allowed?

Reply:
This is needed for jenkins testing to pass (in case there is a fallback due to not being able to allocate enough shared memory, we do not want any error message to be printed in automated testing). Manual testing confirms that the test is getting run.